site/learn/Learn-Queries.md

```diff
@@ -124,6 +124,34 @@ fragment comparisonFields on Character {
 
 You can see how the above query would be pretty repetitive if the fields were repeated. The concept of fragments is frequently used to split complicated application data requirements into smaller chunks, especially when you need to combine lots of UI components with different fragments into one initial data fetch.
 
+### Fragments with arguments
```

Comment:
Could we call this "Using variables inside fragments"? There's an important distinction between arguments and variables.

Reply:
That's a good point. I'll update.